pkg/apiserver/rest/metrics.go

```diff
@@ -10,43 +10,10 @@ import (
 )
 
 type dualWriterMetrics struct {
-	legacy        *prometheus.HistogramVec
-	storage       *prometheus.HistogramVec
```

Comment:
These metrics are nice -- but not required because we already have the key metrics we use to know if things are working:
* k8s HTTP status (the total time)
* grpc stats for unified storage (just the unified time)

This PR drops support for the explicit timing so we can simplify setup and align with where we want to end (eg, unified only everywhere)